pkg/kubernetes/client.js

```diff
@@ -701,6 +727,25 @@ define([
             return connected;
         };
 
+        /**
+         * remove:
+         * @link: a kubernetes item, or selfLink path
+         *
+         * Remove the item from Kubernetes.
+         *
+         * Returns a promise.
+         */
+        self.set_namespace = function set_namespace(namespace) {
```

Comment:
Could we call this ```namespace()``` and if it has an argument, it changes the namespace, if no argument it returns the namespace?